pyproject.toml

```diff
@@ -2,16 +2,44 @@
 build-backend = "poetry.core.masonry.api"
 requires = ["poetry-core"]
 
+[project]
+
+[project.urls]
+Repository = "https://github.com/CORDEX-be2/ValEnsPy"
+Issues = "https://github.com/CORDEX-be2/ValEnsPy/issues"
+Documentation = "https://cordex-be2.github.io/ValEnsPy/"
+Changelog = "https://github.com/CORDEX-be2/ValEnsPy/releases"
+
 [tool.poetry]
+name = "ValEnsPy"
+version = "0.2.0a0"
 authors = [
   "Kobe Vandelanotte <kobe.vandelanotte@meteo.be>",
-  " ... "
-]
+  "Inne Vanderkelen <inne.vanderkelen@kuleuven.be"
+  ]
+maintainers = [
+  "Kobe Vandelanotte <kobe.vandelanotte@meteo.be>",
+  "Inne Vanderkelen <inne.vanderkelen@kuleuven.be"
+  ]
 description = "A Python package to validate gridded model data."
-license = ","
-name = "valenspy"
 readme = "README.md"
-version = "0.1.0"
+license = "GPL-3.0"
+keywords = ["validation", "ensemble", "climate", "meteorology", "model", "data"]
+
+classifiers = [
+  "Development Status :: 3 - Alpha",
```

Comment:
Alpha phase is a mature state of the software. To me it seems best to start in Beta, but @kobebryant432  you known this package best, so this is okay for me.

Reply:
I agree it is not yet mature but from my understanding alpha is before beta? But i could be wrong here.
This is from the https://pypi.org/classifiers/ :

[Development Status :: 1 - Planning](https://pypi.org/search/?c=Development+Status+%3A%3A+1+-+Planning)  
 [Development Status :: 2 - Pre-Alpha](https://pypi.org/search/?c=Development+Status+%3A%3A+2+-+Pre-Alpha) 
  [Development Status :: 3 - Alpha](https://pypi.org/search/?c=Development+Status+%3A%3A+3+-+Alpha)   
[Development Status :: 4 - Beta](https://pypi.org/search/?c=Development+Status+%3A%3A+4+-+Beta)
   [Development Status :: 5 - Production/Stable](https://pypi.org/search/?c=Development+Status+%3A%3A+5+-+Production%2FStable) 
  [Development Status :: 6 - Mature](https://pypi.org/search/?c=Development+Status+%3A%3A+6+-+Mature)  
[Development Status :: 7 - Inactive](https://pypi.org/search/?c=Development+Status+%3A%3A+7+-+Inactive)